src/commcare_cloud/events.py

```diff
@@ -19,7 +19,7 @@ def publish_deploy_event(name, component, environment):
     data = json.dumps({
         "event_type": name,
         "client_payload": {
-            "component": component,
+            "components": components,
```

Comment:
this is a breaking change but the 'component' portion of the payload is not currently used anywhere

FYI @kiran-tripathy to confirm

Reply:
didn't find it being used anywhere